lib/suspenders/app_builder.rb

```diff
@@ -33,7 +33,7 @@ def provide_setup_script
     end
 
     def provide_dev_prime_task
-      copy_file 'development_seeds.rb', 'lib/tasks/development_seeds.rake'
+      copy_file 'development_data.rb', 'lib/tasks/development_data.rake'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.